pkgs/tools/text/mdcat/default.nix

```diff
@@ -2,19 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "mdcat";
-  version = "0.18.2";
+  version = "0.18.4";
 
   src = fetchFromGitHub {
     owner = "lunaryorn";
     repo = pname;
     rev = "mdcat-${version}";
-    sha256 = "0rbfx7951ascgd16jx0d9vcr46ca1v040dc0kfcdfisr6s9ifygw";
+    hash = "sha256-ifNKRZqG+Sy1q0xeHhA+Vl8EYnSyM1WbRmVEnCZgfjk=";
```

Comment:
Are we OK with these now?
https://github.com/NixOS/nixpkgs/pull/89308

Reply:
I'm not sure - I'm using `nixFlakes` locally and that's what it outputs - not sure how to change that.

Reply:
last I heard it is now fine